### ISSUE TITLE
Shrinking flicker still shows when switching between different views

### ISSUE THREAD

**Issue by KevinMehrabi** [StableTechnologies/TEZEX#181]
When switching between Swap, to Liquidity, and within Liquidity switching from Add Liquidity to Remove Liquidity, the screen goes to the shrinking flickering of a condensed desktop view before showing the proper mobile view. See video. 

Also it's not immediately obvious from the video because the opened hamburger menu is blocking the view, but notice the same thing happens when going from liquidity view to swap view. 

Basically every time it changes to a different view, the flickering happens. It goes to a desktop view for half a second before registering it needs to be in mobile view. 

https://github.com/StableTechnologies/TEZEX/assets/4044328/2b500f64-5c65-4ad9-827c-8a7585f15049

**Comment by SiddharthV1** [comment]
On my mobile this happens only initially after loading each component (i.e. when we click on them first) and on subsequent clicks this doesn't happen . can you confirm if this is the behavior for you as well ... depending on that I'll map a strategy

**Comment by KevinMehrabi** [comment]
Resolved by #226 through #229. The mobile trading workspace now transitions continuously between views without the legacy condensed-layout flicker.
